src/main/java/com/github/monster860/fastdmm/objtree/ObjectTree.java

```diff
@@ -476,7 +477,7 @@ public String toString() {
 		return dmePath;
 	}
 
-	/*
+	/**
```

Comment:
This one is technically needed because right now it's not seen as valid Javadoc, and I thought I'd fix it (which I would have done had @monster860 not speed merged that PR)